doc/tutorials/highlevel-bindings.md

```diff
@@ -54,6 +54,34 @@ sufficient to write (or generate) a one-to-one mirror of the C API and use that
 The generated code should still be understandable. So if writing a more idiomatic API on top of the
 direct mapping, significantly simplifies the generated code (and maybe also the template), you should write such an API.
 
+### Bindings for the `lowlevel` Library
+
+Your programming language of choice must provide a way to call into C code (like [cgo](https://golang.org/cmd/cgo/)).
+
+In general we prefer (in this order):
+1. Automatically and statically generated bindings (like [swig](http://www.swig.org/)).
+1. Manually written bindings that directly call to library without any statically build part (like JNI and GI). They are a bit slower but can directly access libelektra.so.
+1. Manually written bindings that are built statically.
+
+If you want to manually write a binding make sure you have a good understanding of the possible limitations the interop layer can have (e.g. variadic functions).
```

Comment:
I think this is generally valid and not only a problem for manually written bindings.

Which other problems might there be next to the varadic functions?